src/Gamecube_Keyboard.h

```diff
@@ -0,0 +1,97 @@
+#ifndef GCKEYMAP_H_INCLUDED
+#define GCKEYMAP_H_INCLUDED
```

Comment:
Could you please add the license info here and the include guard instead of the define statements?